Exercise_1.py

```diff
@@ -1,24 +1,48 @@
+# Time Complexity : O(n) because of push operation and o(1) otherwise
```

Comment:
push in stack is O(1)

Reply:
Hello Ishika! Yes, but for the worst case scenario for array implementation, isn't it O(n). 
Refer: https://iq.opengenus.org/time-complexity-of-stack/ You can correct me if I am wrong.